web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
After colon it sets a default value. In this case empty string. I'm just wondering if this is the way we should be setting default values for spring or if we should do it through `GlobalProperties.java`.